README.md

```diff
@@ -68,6 +68,37 @@ DJANGO_DEV=1 python manage.py test vulnerabilities/tests
 DJANGO_DEV=1 python manage.py import --all
 ```
 
+If you want to run the import periodically, you can use a systemd timer:
```

Comment:
I am not sure why this (which is from #124 )  ends up in your PR.... di you merge rather than rebase on develop?